README.md

```diff
@@ -7,33 +7,59 @@ https://storage.googleapis.com/download.tensorflow.org/models/tflite_11_05_08/re
 
 To validate TensorFlow model on one image run:
 ```bash
-python demo_classification_tf.py --graph /path/to/resnet_v2_101_299_frozen.pb --image /path/to/example.jpeg
+python3 demo_classification.py --graph /path/to/resnet_v2_101_299_frozen.pb --image /path/to/example.jpeg --engine tf
 ```
 
 To validate TensorFlow model on ImageNet dataset run:
 ```bash
-python3 evaluate_tf.py --graph /path/to/resnet_v2_101_299_frozen.pb --dataset /path/to/ILSVRC2012_img_val
+python3 evaluate.py --graph /path/to/resnet_v2_101_299_frozen.pb --dataset /path/to/ILSVRC2012_img_val --engine tf
 ```
 Estimated accuracy:
 ```bash
 Top 1 accuracy: 0.69054
 Top 5 accuracy: 0.89814
 ```
 
-# openvino_quantization
+# openvino_sample
 
-To initialise OpenVINO environment variables open the Command Prompt, and run the setupvars.bat batch file:
+Run the ```convert.py``` script before converting the model, it creates resnet_v2_101_299_opt.pb file in current directory:
 ``bash
-cd C:\Program Files (x86)\IntelSWTools\openvino\bin\
-setupvars.bat
+python3 convert.py --graph resnet_v2_101_299_frozen.pb
+``
+
+To convert TensorFlow model to Intermediate Representation:
+``bash
+cd folder/for/IR/model
```

Comment:
Not needed because user can specify full path by arguments

Reply:
```[ ERROR ]  Both --input_model and --saved_model_dir are defined. Specify either input model or saved model directory.```

Reply:
I mean we don't need to `cd` to the folder with model - user can specify the path by `--input_model`